### PR TITLE
Add template to readme

### PR DESCRIPTION
**Pull request by leninpaulino**
Remove setup steps from readme and add usage, contributing and license section to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,5 +45,21 @@ php artisan db:seed
 
 ```
 
-Now you can start collaborating with Dashi. :)
+## Usage
+
+In order to start using Dashi in your daily routine, you need to: 
+
+1. Generate a Slack Incoming WebHook and copy the URL.
```
Comment (DannyFeliz) [inline README.md]: I think that the step 1 should be a link like:
1. Generate a [Slack Incoming WebHooks](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks) and copy the URL.

**Review by DannyFeliz — CHANGES_REQUESTED** [review]

**Comment by leninpaulino** [comment]
@DannyFeliz done.

**Review by leninpaulino — COMMENTED** [review]
Of course, I forget it.

**Comment by gabbanaesteban** [comment]
resolve #50

**Pull request merged**